packages/@aws-cdk/aws-apprunner/lib/observability-configuration.ts

```diff
@@ -0,0 +1,126 @@
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnObservabilityConfiguration } from './apprunner.generated';
+
+/**
+ * Properties of the AppRunner Observability Configuration
+ */
+export interface ObservabilityConfigurationProps {
```

Comment:
Also, all fields throughout this code change should specify public/private/protected.